scripts/manage_study.py

```diff
@@ -140,18 +140,24 @@ def download_from_bucket(file_path):
     return destination
 
 def validate_metadata_file(metadata_path):
-    placeholder = 'SCP555' # TODO: Refactor CellMetadata to not require this
-    metadata = CellMetadata(metadata_path, '', '', study_accession=placeholder)
-    print(f'Validating {metadata_path}')
-
-    convention_path = download_from_bucket(IngestPipeline.JSON_CONVENTION)
-    with open(convention_path) as file:
-        convention = json.load(file)
-
-    validate_input_metadata(metadata, convention)
-    serialize_issues(metadata)
-    report_issues(metadata)
-    exit_if_errors(metadata)
+    study_accession_res =connection.get_study_attribute(
+        study_name=parsed_args.study_name,
+        attribute='accession',
+        dry_run=parsed_args.dry_run)
+    if succeeded(study_accession_res):
+        if verbose:
+            print(f'Study accession {study_accession_res} retrieved for {parsed_args.study_name}')
+        study_accession = study_accession_res.get('study_attribute')
+        metadata = CellMetadata(metadata_path, '', '', study_accession=str(study_accession))
+        convention_res = connection.do_get(command=api_base + 'metadata_schemas/alexandria_convention/latest/json',dry_run=parsed_args.dry_run)
+        if succeeded(convention_res ):
+            if verbose:
+                print(f'Retreieved file for latest metdata convention')
+            convention = convention_res["response"].json()
+            validate_input_metadata(metadata, convention)
```

Comment:
We want to make sure the ingest pipeline code the user has locally is compatible (preferably the same) as the code the validation server (typically production) is using. I'm not 100% sure how we can make sure their local python install of scp-ingest-pipeline is in sync with the version in the docker container that the validation server is using.
 
We could check, at least, that the version of the metadata convention used by ingest pipeline (which should be parsable from IngestPipeline.JSON_CONVENTION) matches the version number of the metadata convention that was retrieved by API (the version number is parsable from the value for $id in the convention json). 

If the conventions versions don't match, the logic/code being run for validation will definitely not match and the user should be prompted to action (for validation against production, SCP should be keeping requirements.txt and setup.py in this repo up to date so it is valid against production for manage_study.py) to git pull against this repo and pip install -r requirements.txt. 

If there are other, less hack-y solutions to ensure that the user is running the appropriate version of ingest code, that would be even better.

Reply:
@eweitz I remember speaking with you about this issue. Do you mind commenting on this.

Reply:
Per our group chat just now, I think we've got a plan to address the issues Jean notes.

Reply:
[SCP-2415](https://broadworkbench.atlassian.net/browse/SCP-2415) and [SCP-2414](https://broadworkbench.atlassian.net/browse/SCP-2414) track these issues